internal/commands/pull/inputs_test.go

```diff
@@ -188,7 +188,7 @@ func TestPullTemplatesResolve(t *testing.T) {
 
 		_, ui := mock.NewUI()
 		input := inputs{}
-		_, err := input.resolveClientTemplates(ui, realmClient, "some-group-id", "some-app-id")
+		_, err = input.resolveClientTemplates(ui, realmClient, "some-group-id", "some-app-id")
```

Comment:
all of these changes seem to be able to be reverted (they're shadowing `err` from higher-in-scope tests at this point)